src/app/app.module.ts

```diff
@@ -100,5 +100,6 @@ import {PoseComponent} from "./joint-control/pose/pose.component";
         },
     ],
     bootstrap: [AppComponent],
+    // entryComponents: [MatDialogModule],
```

Comment:
do we need this?

Reply:
Due to the migration to Angular 18, not anymore. I removed the comment.